### ISSUE TITLE
Upgrade from 1.3.2 to 1.4.3 breaks the build

### ISSUE THREAD

**Issue by draganjovanovic1** [transistorsoft/flutter_background_geolocation#162]
After upgrading the plugin from 1.3.2 to 1.4.3 I can't compile project any more. I get this error:
```
Note: /Users/dragan/Library/flutter/.pub-cache/hosted/pub.dartlang.org/background_fetch-0.3.2/android/src/main/java/com/transistorsoft/flutter/backgroundfetch/HeadlessTask.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.                    
Note: /Users/dragan/Library/flutter/.pub-cache/hosted/pub.dartlang.org/device_info-0.4.1+2/android/src/main/java/io/flutter/plugins/deviceinfo/DeviceInfoPlugin.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.                    
Note: /Users/dragan/Library/flutter/.pub-cache/hosted/pub.dartlang.org/package_info-0.4.0+11/android/src/main/java/io/flutter/plugins/packageinfo/PackageInfoPlugin.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.                    
Note: /Users/dragan/Library/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_core-0.4.2+1/android/src/main/java/io/flutter/plugins/firebase/core/FirebaseCorePlugin.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.                    
Note: /Users/dragan/Library/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_background_geolocation-1.4.3/android/src/main/java/com/transistorsoft/flutter/backgroundgeolocation/HeadlessTask.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.                    
Note: /Users/dragan/Library/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_background_geolocation-1.4.3/android/src/main/java/com/transistorsoft/flutter/backgroundgeolocation/FLTBackgroundGeolocationPlugin.java uses unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.                      
Note: /Users/dragan/Library/flutter/.pub-cache/hosted/pub.dartlang.org/path_provider-1.4.5/android/src/main/java/io/flutter/plugins/pathprovider/PathProviderPlugin.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.                    
Note: /Users/dragan/Library/flutter/.pub-cache/hosted/pub.dartlang.org/permission_handler-3.3.0/android/src/main/java/com/baseflow/permissionhandler/PermissionHandlerPlugin.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.                    
/Users/dragan/Library/flutter/.pub-cache/hosted/pub.dartlang.org/shared_preferences-0.5.6/android/src/main/java/io/flutter/plugins/sharedpreferences/SharedPreferencesPlugin.java:25: warning: [deprecation] getFlutterEngine() in FlutterPluginBinding has been deprecated
    setupChannel(binding.getFlutterEngine().getDartExecutor(), binding.getApplicationContext());
                        ^                                               
1 warning                                                               
D8: Program type already present: com.transistorsoft.locationmanager.BootReceiver
                                                                        
FAILURE: Build failed with an exception.                                
                                                                        
* What went wrong:                                                      
Execution failed for task ':app:transformDexArchiveWithExternalLibsDexMergerForDebug'.
> com.android.builder.dexing.DexArchiveMergerException: Error while merging dex archives: 
  Learn how to resolve the issue at https://developer.android.com/studio/build/dependencies#duplicate_classes.
  Program type already present: com.transistorsoft.locationmanager.BootReceiver
                                                                        
* Try:                                                                  
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
                                                                        
* Get more help at https://help.gradle.org                              
                                                                        
BUILD FAILED in 27s                                                     
Running Gradle task 'assembleDebug'...                                  
Running Gradle task 'assembleDebug'... Done                        28.6s
Gradle task assembleDebug failed with exit code 1
```
I have followed step by step instructions to install the plugin and had no issues installing it initially. It worked perfectly since the beginning but now I have problem updating it. 

I tried deleting all cached build folders, `flutter clean`, `flutter pub cache repair` but nothing helped. Please advise.

## Your Environment
* Plugin version: 1.4.3
* Platform: Android
* OS version: macOS Catalina 10.15.1
* Device manufacturer / model: 
* Flutter info (`flutter doctor`):
```
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel master, v1.13.1-pre.99, on Mac OS X 10.15.1 19B88, locale en-RS)
 
[✓] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
[✓] Xcode - develop for iOS and macOS (Xcode 11.2.1)
[✓] Android Studio (version 3.5)
[✓] VS Code (version 1.40.2)
[✓] Connected device (1 available)

• No issues found!
```
* Plugin config:
```dart <-- Syntax highlighting: DO NOT REMOVE -->
bg.BackgroundGeolocation.ready(bg.Config(
            desiredAccuracy: bg.Config.DESIRED_ACCURACY_HIGH,
            distanceFilter: 10.0,
            locationUpdateInterval: 30000,
            maxRecordsToPersist: 50,
            disableElasticity: false,
            enableHeadless: true,
            foregroundService: true,
            stopOnTerminate: false,
            startOnBoot: true,
            preventSuspend: true,
            autoSync: false,
            maxDaysToPersist: 5,
            allowIdenticalLocations: false,
            debug: false,
            logLevel: bg.Config.LOG_LEVEL_VERBOSE,
            notification:
                bg.Notification(smallIcon: 'mipmap/ic_stat_notification_icon')))
        .then((bg.State state) {
      if (!state.enabled) {
        bg.BackgroundGeolocation.start();
      }
    });
```

## Expected Behavior
Build should pass

## Actual Behavior
Build fails

**Comment by christocracy** [comment]
clean your build

**Comment by draganjovanovic1** [comment]
I have tried it. 

_I tried deleting all cached build folders, flutter clean, flutter pub cache repair but nothing helped. Please advise._

**Comment by draganjovanovic1** [comment]
If you have some other idea to try or need any additional information from me please let me know.

**Comment by christocracy** [comment]
1.4.3 is corrupt.  I messed up and uploaded 2 copies of the Android library `tslocationmanager.aar`.  Upgrade to [`1.4.4`, just published](https://pub.dev/packages/flutter_background_geolocation/versions)

**Comment by draganjovanovic1** [comment]
Tested, Andriod works nice. Thanks.

However, there is now an issue with iOS:
```
 
Launching lib/main.dart on iPhone 11 Pro Max in debug mode...
 
Running pod install...                                              1.4s
CocoaPods' output:
↳
      Preparing

    Analyzing dependencies

    Inspecting targets to integrate
      Using `ARCHS` setting to build architectures of target `Pods-Runner`: (``)

    Finding Podfile changes
      A firebase_core_web
      - Firebase
      - Flutter
      - background_fetch
      - catcher
      - device_info
      - firebase_auth
      - firebase_auth_web
      - firebase_core
      - flutter_background_geolocation
      - flutter_local_notifications
      - flutter_mailer
      - fluttertoast
      - package_info
      - path_provider
      - permission_handler
      - shared_preferences
      - shared_preferences_macos
      - shared_preferences_web
      - wakelock

    Fetching external sources
    -> Fetching podspec for `Flutter` from `Flutter`
    -> Fetching podspec for `background_fetch` from `.symlinks/plugins/background_fetch/ios`
    -> Fetching podspec for `catcher` from `.symlinks/plugins/catcher/ios`
    -> Fetching podspec for `device_info` from `.symlinks/plugins/device_info/ios`
    -> Fetching podspec for `firebase_auth` from `.symlinks/plugins/firebase_auth/ios`
    -> Fetching podspec for `firebase_auth_web` from `.symlinks/plugins/firebase_auth_web/ios`
    -> Fetching podspec for `firebase_core` from `.symlinks/plugins/firebase_core/ios`
    -> Fetching podspec for `firebase_core_web` from `.symlinks/plugins/firebase_core_web/ios`
    -> Fetching podspec for `flutter_background_geolocation` from `.symlinks/plugins/flutter_background_geolocation/ios`
    -> Fetching podspec for `flutter_local_notifications` from `.symlinks/plugins/flutter_local_notifications/ios`
    -> Fetching podspec for `flutter_mailer` from `.symlinks/plugins/flutter_mailer/ios`
    -> Fetching podspec for `fluttertoast` from `.symlinks/plugins/fluttertoast/ios`
    -> Fetching podspec for `package_info` from `.symlinks/plugins/package_info/ios`
    -> Fetching podspec for `path_provider` from `.symlinks/plugins/path_provider/ios`
    -> Fetching podspec for `permission_handler` from `.symlinks/plugins/permission_handler/ios`
    -> Fetching podspec for `shared_preferences` from `.symlinks/plugins/shared_preferences/ios`
    -> Fetching podspec for `shared_preferences_macos` from `.symlinks/plugins/shared_preferences_macos/ios`
    -> Fetching podspec for `shared_preferences_web` from `.symlinks/plugins/shared_preferences_web/ios`
    -> Fetching podspec for `wakelock` from `.symlinks/plugins/wakelock/ios`

    Resolving dependencies of `Podfile`
      CDN: trunk Relative path: CocoaPods-version.yml exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: all_pods_versions_0_3_5.txt exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/0/3/5/Firebase/6.13.0/Firebase.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/0/3/5/Firebase/6.13.0/Firebase.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/0/3/5/Firebase/6.13.0/Firebase.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/0/3/5/Firebase/6.13.0/Firebase.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: all_pods_versions_d_1_8.txt exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/d/1/8/CocoaLumberjack/3.6.0/CocoaLumberjack.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/0/3/5/Firebase/6.13.0/Firebase.podspec.json exists! Returning local because checking is only perfomed in repo update
    [!] CocoaPods could not find compatible versions for pod "CocoaLumberjack":
      In snapshot (Podfile.lock):
        CocoaLumberjack (= 3.6.0, ~> 3.0)

      In Podfile:
        flutter_background_geolocation (from `.symlinks/plugins/flutter_background_geolocation/ios`) was resolved to 0.2.0, which depends on
          CocoaLumberjack (~> 3.5.0)


    You have either:
     * out-of-date source repos which you can update with `pod repo update` or with `pod install --repo-update`.
     * changed the constraints of dependency `CocoaLumberjack` inside your development pod `flutter_background_geolocation`.
       You should run `pod update CocoaLumberjack` to apply changes you've made.

    /usr/local/Cellar/cocoapods/1.8.4/libexec/gems/molinillo-0.6.6/lib/molinillo/resolution.rb:328:in `raise_error_unless_state'
    /usr/local/Cellar/cocoapods/1.8.4/libexec/gems/molinillo-0.6.6/lib/molinillo/resolution.rb:310:in `block in unwind_for_conflict'
    /usr/local/Cellar/cocoapods/1.8.4/libexec/gems/molinillo-0.6.6/lib/molinillo/resolution.rb:308:in `tap'
    /usr/local/Cellar/cocoapods/1.8.4/libexec/gems/molinillo-0.6.6/lib/molinillo/resolution.rb:308:in `unwind_for_conflict'
    /usr/local/Cellar/cocoapods/1.8.4/libexec/gems/molinillo-0.6.6/lib/molinillo/resolution.rb:257:in `process_topmost_state'
    /usr/local/Cellar/cocoapods/1.8.4/libexec/gems/molinillo-0.6.6/lib/molinillo/resolution.rb:182:in `resolve'
    /usr/local/Cellar/cocoapods/1.8.4/libexec/gems/molinillo-0.6.6/lib/molinillo/resolver.rb:43:in `resolve'
    /usr/local/Cellar/cocoapods/1.8.4/libexec/gems/cocoapods-1.8.4/lib/cocoapods/resolver.rb:94:in `resolve'
    /usr/local/Cellar/cocoapods/1.8.4/libexec/gems/cocoapods-1.8.4/lib/cocoapods/installer/analyzer.rb:986:in `block in resolve_dependencies'
    /usr/local/Cellar/cocoapods/1.8.4/libexec/gems/cocoapods-1.8.4/lib/cocoapods/user_interface.rb:64:in `section'
    /usr/local/Cellar/cocoapods/1.8.4/libexec/gems/cocoapods-1.8.4/lib/cocoapods/installer/analyzer.rb:984:in `resolve_dependencies'
    /usr/local/Cellar/cocoapods/1.8.4/libexec/gems/cocoapods-1.8.4/lib/cocoapods/installer/analyzer.rb:124:in `analyze'
    /usr/local/Cellar/cocoapods/1.8.4/libexec/gems/cocoapods-1.8.4/lib/cocoapods/installer.rb:410:in `analyze'
    /usr/local/Cellar/cocoapods/1.8.4/libexec/gems/cocoapods-1.8.4/lib/cocoapods/installer.rb:234:in `block in resolve_dependencies'
    /usr/local/Cellar/cocoapods/1.8.4/libexec/gems/cocoapods-1.8.4/lib/cocoapods/user_interface.rb:64:in `section'
    /usr/local/Cellar/cocoapods/1.8.4/libexec/gems/cocoapods-1.8.4/lib/cocoapods/installer.rb:233:in `resolve_dependencies'
    /usr/local/Cellar/cocoapods/1.8.4/libexec/gems/cocoapods-1.8.4/lib/cocoapods/installer.rb:156:in `install!'
    /usr/local/Cellar/cocoapods/1.8.4/libexec/gems/cocoapods-1.8.4/lib/cocoapods/command/install.rb:52:in `run'
    /usr/local/Cellar/cocoapods/1.8.4/libexec/gems/claide-1.0.3/lib/claide/command.rb:334:in `run'
    /usr/local/Cellar/cocoapods/1.8.4/libexec/gems/cocoapods-1.8.4/lib/cocoapods/command.rb:52:in `run'
    /usr/local/Cellar/cocoapods/1.8.4/libexec/gems/cocoapods-1.8.4/bin/pod:55:in `<top (required)>'
    /usr/local/Cellar/cocoapods/1.8.4/libexec/bin/pod:23:in `load'
    /usr/local/Cellar/cocoapods/1.8.4/libexec/bin/pod:23:in `<main>'

Error output from CocoaPods:
↳

    [!] Automatically assigning platform `iOS` with version `8.0` on target `Runner` because no platform was specified. Please specify a platform for this target in your Podfile. See
    `https://guides.cocoapods.org/syntax/podfile.html#platform`.

Error: CocoaPods's specs repository is too out-of-date to satisfy dependencies.
To update the CocoaPods specs, run:
  pod repo update



Error running pod install
Error launching application on iPhone 11 Pro Max.
```

Tried to run `pod repo update` but no success to resolve the issue.

Per my understanding the issue is with `flutter_background_geolocation` package. 

```
[!] CocoaPods could not find compatible versions for pod "CocoaLumberjack":
  In snapshot (Podfile.lock):
    CocoaLumberjack (= 3.6.0, ~> 3.0)

  In Podfile:
    flutter_background_geolocation (from `.symlinks/plugins/flutter_background_geolocation/ios`) was resolved to 0.2.0, which depends on
      CocoaLumberjack (~> 3.5.0)


You have either:
 * changed the constraints of dependency `CocoaLumberjack` inside your development pod `flutter_background_geolocation`.
   You should run `pod update CocoaLumberjack` to apply changes you've made.
```

Please advise.

**Comment by christocracy** [comment]
Some other plugin out there is using CococaLumberjack beside me.

Install from Git:

```
dependencies:
  flutter_background_geolocation:
    git:
      url: https://github.com/transistorsoft/flutter_background_geolocation.git
      ref: dev
```

**Comment by draganjovanovic1** [comment]
Will try git dependency.

Here is my `Podfile.lock` in case you can tell something from it. 

```
PODS:
  - background_fetch (0.1.1):
    - Flutter
  - catcher (0.0.1):
    - Flutter
  - CocoaLumberjack (3.6.0):
    - CocoaLumberjack/Core (= 3.6.0)
  - CocoaLumberjack/Core (3.6.0)
  - device_info (0.0.1):
    - Flutter
  - Firebase/Analytics (6.13.0):
    - Firebase/Core
  - Firebase/Auth (6.13.0):
    - Firebase/CoreOnly
    - FirebaseAuth (~> 6.4.0)
  - Firebase/Core (6.13.0):
    - Firebase/CoreOnly
    - FirebaseAnalytics (= 6.1.6)
  - Firebase/CoreOnly (6.13.0):
    - FirebaseCore (= 6.4.0)
  - firebase_auth (0.0.1):
    - Firebase/Auth (~> 6.0)
    - Firebase/Core
    - Flutter
  - firebase_auth_web (0.1.0):
    - Flutter
  - firebase_core (0.0.1):
    - Firebase/Core
    - Flutter
  - FirebaseAnalytics (6.1.6):
    - FirebaseCore (~> 6.4)
    - FirebaseInstanceID (~> 4.2)
    - GoogleAppMeasurement (= 6.1.6)
    - GoogleUtilities/AppDelegateSwizzler (~> 6.0)
    - GoogleUtilities/MethodSwizzler (~> 6.0)
    - GoogleUtilities/Network (~> 6.0)
    - "GoogleUtilities/NSData+zlib (~> 6.0)"
    - nanopb (= 0.3.9011)
  - FirebaseAuth (6.4.0):
    - FirebaseAuthInterop (~> 1.0)
    - FirebaseCore (~> 6.2)
    - GoogleUtilities/AppDelegateSwizzler (~> 6.2)
    - GoogleUtilities/Environment (~> 6.2)
    - GTMSessionFetcher/Core (~> 1.1)
  - FirebaseAuthInterop (1.0.0)
  - FirebaseCore (6.4.0):
    - FirebaseCoreDiagnostics (~> 1.0)
    - FirebaseCoreDiagnosticsInterop (~> 1.0)
    - GoogleUtilities/Environment (~> 6.2)
    - GoogleUtilities/Logger (~> 6.2)
  - FirebaseCoreDiagnostics (1.1.2):
    - FirebaseCoreDiagnosticsInterop (~> 1.0)
    - GoogleDataTransportCCTSupport (~> 1.0)
    - GoogleUtilities/Environment (~> 6.2)
    - GoogleUtilities/Logger (~> 6.2)
    - nanopb (~> 0.3.901)
  - FirebaseCoreDiagnosticsInterop (1.1.0)
  - FirebaseInstanceID (4.2.7):
    - FirebaseCore (~> 6.0)
    - GoogleUtilities/Environment (~> 6.0)
    - GoogleUtilities/UserDefaults (~> 6.0)
  - Flutter (1.0.0)
  - flutter_background_geolocation (0.2.0):
    - CocoaLumberjack (~> 3.0)
    - Flutter
  - flutter_local_notifications (0.0.1):
    - Flutter
  - flutter_mailer (0.0.1):
    - Flutter
  - fluttertoast (0.0.2):
    - Flutter
  - GoogleAppMeasurement (6.1.6):
    - GoogleUtilities/AppDelegateSwizzler (~> 6.0)
    - GoogleUtilities/MethodSwizzler (~> 6.0)
    - GoogleUtilities/Network (~> 6.0)
    - "GoogleUtilities/NSData+zlib (~> 6.0)"
    - nanopb (= 0.3.9011)
  - GoogleDataTransport (3.2.0)
  - GoogleDataTransportCCTSupport (1.2.2):
    - GoogleDataTransport (~> 3.2)
    - nanopb (~> 0.3.901)
  - GoogleUtilities/AppDelegateSwizzler (6.3.2):
    - GoogleUtilities/Environment
    - GoogleUtilities/Logger
    - GoogleUtilities/Network
  - GoogleUtilities/Environment (6.3.2)
  - GoogleUtilities/Logger (6.3.2):
    - GoogleUtilities/Environment
  - GoogleUtilities/MethodSwizzler (6.3.2):
    - GoogleUtilities/Logger
  - GoogleUtilities/Network (6.3.2):
    - GoogleUtilities/Logger
    - "GoogleUtilities/NSData+zlib"
    - GoogleUtilities/Reachability
  - "GoogleUtilities/NSData+zlib (6.3.2)"
  - GoogleUtilities/Reachability (6.3.2):
    - GoogleUtilities/Logger
  - GoogleUtilities/UserDefaults (6.3.2):
    - GoogleUtilities/Logger
  - GTMSessionFetcher/Core (1.3.0)
  - nanopb (0.3.9011):
    - nanopb/decode (= 0.3.9011)
    - nanopb/encode (= 0.3.9011)
  - nanopb/decode (0.3.9011)
  - nanopb/encode (0.3.9011)
  - package_info (0.0.1):
    - Flutter
  - path_provider (0.0.1):
    - Flutter
  - permission_handler (3.3.0):
    - Flutter
  - shared_preferences (0.0.1):
    - Flutter
  - shared_preferences_macos (0.0.1):
    - Flutter
  - shared_preferences_web (0.0.1):
    - Flutter
  - wakelock (0.0.1):
    - Flutter

DEPENDENCIES:
  - background_fetch (from `.symlinks/plugins/background_fetch/ios`)
  - catcher (from `.symlinks/plugins/catcher/ios`)
  - device_info (from `.symlinks/plugins/device_info/ios`)
  - Firebase/Analytics
  - Firebase/Auth
  - firebase_auth (from `.symlinks/plugins/firebase_auth/ios`)
  - firebase_auth_web (from `.symlinks/plugins/firebase_auth_web/ios`)
  - firebase_core (from `.symlinks/plugins/firebase_core/ios`)
  - Flutter (from `Flutter`)
  - flutter_background_geolocation (from `.symlinks/plugins/flutter_background_geolocation/ios`)
  - flutter_local_notifications (from `.symlinks/plugins/flutter_local_notifications/ios`)
  - flutter_mailer (from `.symlinks/plugins/flutter_mailer/ios`)
  - fluttertoast (from `.symlinks/plugins/fluttertoast/ios`)
  - package_info (from `.symlinks/plugins/package_info/ios`)
  - path_provider (from `.symlinks/plugins/path_provider/ios`)
  - permission_handler (from `.symlinks/plugins/permission_handler/ios`)
  - shared_preferences (from `.symlinks/plugins/shared_preferences/ios`)
  - shared_preferences_macos (from `.symlinks/plugins/shared_preferences_macos/ios`)
  - shared_preferences_web (from `.symlinks/plugins/shared_preferences_web/ios`)
  - wakelock (from `.symlinks/plugins/wakelock/ios`)

SPEC REPOS:
  trunk:
    - CocoaLumberjack
    - Firebase
    - FirebaseAnalytics
    - FirebaseAuth
    - FirebaseAuthInterop
    - FirebaseCore
    - FirebaseCoreDiagnostics
    - FirebaseCoreDiagnosticsInterop
    - FirebaseInstanceID
    - GoogleAppMeasurement
    - GoogleDataTransport
    - GoogleDataTransportCCTSupport
    - GoogleUtilities
    - GTMSessionFetcher
    - nanopb

EXTERNAL SOURCES:
  background_fetch:
    :path: ".symlinks/plugins/background_fetch/ios"
  catcher:
    :path: ".symlinks/plugins/catcher/ios"
  device_info:
    :path: ".symlinks/plugins/device_info/ios"
  firebase_auth:
    :path: ".symlinks/plugins/firebase_auth/ios"
  firebase_auth_web:
    :path: ".symlinks/plugins/firebase_auth_web/ios"
  firebase_core:
    :path: ".symlinks/plugins/firebase_core/ios"
  Flutter:
    :path: Flutter
  flutter_background_geolocation:
    :path: ".symlinks/plugins/flutter_background_geolocation/ios"
  flutter_local_notifications:
    :path: ".symlinks/plugins/flutter_local_notifications/ios"
  flutter_mailer:
    :path: ".symlinks/plugins/flutter_mailer/ios"
  fluttertoast:
    :path: ".symlinks/plugins/fluttertoast/ios"
  package_info:
    :path: ".symlinks/plugins/package_info/ios"
  path_provider:
    :path: ".symlinks/plugins/path_provider/ios"
  permission_handler:
    :path: ".symlinks/plugins/permission_handler/ios"
  shared_preferences:
    :path: ".symlinks/plugins/shared_preferences/ios"
  shared_preferences_macos:
    :path: ".symlinks/plugins/shared_preferences_macos/ios"
  shared_preferences_web:
    :path: ".symlinks/plugins/shared_preferences_web/ios"
  wakelock:
    :path: ".symlinks/plugins/wakelock/ios"

SPEC CHECKSUMS:
  background_fetch: 385552e57d3bde2c7a3abd6b1edc76b87823b6c4
  catcher: e5585ed31458c849f4fd9c3719b2fcb0188b8918
  CocoaLumberjack: 78b0c238666f4f58db069738ec176f4519557516
  device_info: cbf09d2ec12aa7110e0b09fabe54b5bd6c8efe74
  Firebase: 458d109512200d1aca2e1b9b6cf7d68a869a4a46
  firebase_auth: 48dca46d6a22ab8522633c6257715c4ee5afb0f9
  firebase_auth_web: 0955c07bcc06e84af76b9d4e32e6f31518f2d7de
  firebase_core: 15d27cd74b2e40f8c29540707c0486bb561a92e8
  FirebaseAnalytics: 45f36d9c429fc91d206283900ab75390cd05ee8a
  FirebaseAuth: 7d0f84873926f6648bbd1391a318dfb1a26b5e4f
  FirebaseAuthInterop: 0ffa57668be100582bb7643d4fcb7615496c41fc
  FirebaseCore: 307ea2508df730c5865334e41965bd9ea344b0e5
  FirebaseCoreDiagnostics: 511f4f3ed7d440bb69127e8b97c2bc8befae639e
  FirebaseCoreDiagnosticsInterop: e9b1b023157e3a2fc6418b5cb601e79b9af7b3a0
  FirebaseInstanceID: ebd2ea79ee38db0cb5f5167b17a0d387e1cc7b6e
  Flutter: 0e3d915762c693b495b44d77113d4970485de6ec
  flutter_background_geolocation: fc9b54bdcaedea40e324cae19e9d8aac7bc045d6
  flutter_local_notifications: 9e4738ce2471c5af910d961a6b7eadcf57c50186
  flutter_mailer: 2ef5a67087bc8c6c4cefd04a178bf1ae2c94cd83
  fluttertoast: b644586ef3b16f67fae9a1f8754cef6b2d6b634b
  GoogleAppMeasurement: dfe55efa543e899d906309eaaac6ca26d249862f
  GoogleDataTransport: 8e9b210c97d55fbff306cc5468ff91b9cb32dcf5
  GoogleDataTransportCCTSupport: ef79a4728b864946a8aafdbab770d5294faf3b5f
  GoogleUtilities: 547a86735c6f0ee30ad17e94df4fc21f616b71cb
  GTMSessionFetcher: 43b8b64263023d4f32caa0b40f4c8bfa3c5f36d8
  nanopb: 18003b5e52dab79db540fe93fe9579f399bd1ccd
  package_info: 48b108e75b8802c2d5e126f208ef540561c98aef
  path_provider: fb74bd0465e96b594bb3b5088ee4a4e7bb1f2a9d
  permission_handler: 67637977b227d62d46bfbf524f335f8568de5a73
  shared_preferences: 430726339841afefe5142b9c1f50cb6bd7793e01
  shared_preferences_macos: f3f29b71ccbb56bf40c9dd6396c9acf15e214087
  shared_preferences_web: 141cce0c3ed1a1c5bf2a0e44f52d31eeb66e5ea9
  wakelock: da5012e206ec8978df58d8642e4cdd4fe506185f

PODFILE CHECKSUM: 2537432d2fa34e591bb59498629638243780ae28

COCOAPODS: 1.8.4
```

**Comment by christocracy** [comment]
Update me ASAP after installing from Git so I can re-publish.

**Comment by draganjovanovic1** [comment]
@christocracy git dependency worked for both Android and iOS. 👍

**Comment by christocracy** [comment]
Published [1.4.5](https://pub.dev/packages/flutter_background_geolocation/versions/1.4.5)
